src/dmap_import/schemas/__init__.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import DeclarativeBase
+
+
+# pylint: disable=too-few-public-methods
+class SqlBase(DeclarativeBase):
+    """SqlAlchemy Schema Base Class"""
+
+
+# pylint: disable=cyclic-import
+# pylint: disable=wrong-import-position
+from .agg_average_boardings_by_day_type_month import (
+    AggAvgBoardingsByDayTypeMonth,
+)
+from .agg_boardings_fareprod_mode_month import AggBoardingsFareprodModeMonth
+from .agg_daily_fareprod_route import AggDailyFareprodRoute
+from .agg_daily_fareprod_station import AggDailyFareprodStation
+from .agg_daily_fareprod import AggDailyFareprod
+from .agg_daily_transfers_route import AggDailyTransfersRoute
+from .agg_daily_transfers_station import AggDailyTransfersStation
+from .agg_hourly_entry_exit_count import AggHourlyEntryExitCount
+from .agg_total_boardings_month_mode import AggTotalBoardingsMonthMode
+from .device_event import DeviceEvents
+from .sale_transaction import SaleTransaction
+from .use_transaction_location import UseTransactionalLocation
+from .use_transaction_longitudinal import UseTransactionalLongitudinal
```

Comment:
This is the recommended format for saving schema/model information in separate files (which I think we want here). 

pylint complains about circular imports and out of order imports, but alembic can't pick up all of the schemas for auto-migrations without this configuration.